packages/dds/tree/src/simple-tree/objectNode.ts

```diff
@@ -94,6 +94,7 @@ export type FieldHasDefault<T extends ImplicitFieldSchema> = T extends FieldSche
  * Helper used to produce types for:
  *
  * 1. Insertable content which can be used to construct an object node.
+ * In this case, only own properties are considered to reduce the risk of incorrectly interpreting data at the cost of occasionally requiring users to convert data into a compatible format.
```

Comment:
```suggestion
 * In this case, only own properties are considered.
 * This reduces the risk of incorrectly interpreting data at the cost of occasionally requiring users to convert data into a compatible format.
```
The current sentence structure could be read as "only own properties reduce the risk of...", which is true but not the main point.